src/main/resources/db/migration/V3_2__set_frist_null_forhandsvarsel_vurdering.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE AKTIVITETSKRAV_VURDERING
+SET frist = null
```

Comment:
Hvorfor skal disse null'es ut?

Reply:
Fordi den relevante datoen er lagt inn i aktivitetskrav_varsel i stedet.

Reply:
Vi har nå et nytt felt på varsel for svarfrist som inneholder den samme fristen, så tenker det var ryddigst å nulle ut disse.